.github/workflows/semver_testing.yml

```diff
@@ -0,0 +1,27 @@
+### Ensure that our local testing always passes
+name: Run semver tests
```

Comment:
Please quote your yaml strings or [bad things can happen](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell)